rollbar-web/src/main/java/com/rollbar/web/filter/RollbarFilter.java

```diff
@@ -0,0 +1,58 @@
+package com.rollbar.web.filter;
+
+import static com.rollbar.notifier.config.ConfigBuilder.withAccessToken;
+
+import com.rollbar.notifier.Rollbar;
+import com.rollbar.web.provider.PersonProvider;
+import com.rollbar.web.provider.RequestProvider;
+import java.io.IOException;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+
+public class RollbarFilter implements Filter {
+
+  private Rollbar rollbar;
+
+  RollbarFilter(Rollbar rollbar) {
+    this.rollbar = rollbar;
+  }
+
+  @Override
+  public void init(FilterConfig filterConfig) throws ServletException {
+    String accessToken = filterConfig.getInitParameter("access_token");
+
+    rollbar = Rollbar.init(withAccessToken(accessToken)
+        .language("java")
```

Comment:
This should be the default and shouldn't need to be specified.

Reply:
Sure, this will be set by default by the `ConfigBuilder` located at the module `rollbar-java`, but as at this moment it has not default values assigned I just did it here for testing purposes.